File: SampleSetup/Output/Swift/Model/ApiCountry.swift
    struct ApiCountry: Codable, AbstractModelWithId
 {
        var TitleShort: String

        var TitleLong: String

        var PhoneCode: String

        var Code: String

        var Cities: [ApiCity] = [ApiCity]()

        var Addresses: [ApiAddress] = [ApiAddress]()

    }

File: SampleSetup/Output/PHP/SampleProject/Protocol/Model/ApiCountry.php
<?php
namespace SampleProject\Protocol\Model;

class ApiCountry extends \SampleProject\Protocol\Model\AbstractModelWithId {
    /** @var string */ public $TitleShort;

    /** @var string */ public $TitleLong;

    /** @var string */ public $PhoneCode;

    /** @var string */ public $Code;

    /** @var \SampleProject\Protocol\Model\ApiCity[] */ public $Cities;

    /** @var \SampleProject\Protocol\Model\ApiAddress[] */ public $Addresses;

    public function __construct()
    {
        parent::__construct();
        $this->Cities = [];
        $this->Addresses = [];
    }

}
